schedule/security/cc_netfilter_netfilebt.yaml

```diff
@@ -7,7 +7,7 @@ schedule:
     - security/selinux/selinux_setup
     - security/cc/cc_selinux_setup
     - security/cc/cc_audit_test_setup
-    - security/cc/netfilter
+    - security/cc/setup_env
```

Comment:
The name looks not that easy to figure out which cases are using for testing. If this is for Netfilter and Netfilebt testing, perhaps we can use i.g. `setup_cc_net_env`  or `setup_net_test_env`  ...etc. How do you think?